### ISSUE TITLE
Accessibility issue: Multiple role="navigation" tags for nav menu widget

### ISSUE THREAD

**Issue by besweeet** [elementor/elementor#11372]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
When a nav menu widget is being used, there are multiple `role="navigation"` tags. Per WCAG guidelines, this is okay, but they would each need a unique label, which doesn't appear to be possible to add. Ideally, there would just be one `role="navigation"` tag. Why are there two?
